apps/frontend/components/field/referee/prestart.tsx

```diff
@@ -48,6 +57,10 @@ const MatchPrestart: React.FC<MatchPrestartProps> = ({
             </Typography>
           )}
 
+          {inspectionStartTime && participant.present === 'present' && (
```

Comment:
don't need to check for present here since inspection start time inherently check that